tools/risk-form-filler/src/object-models/sections.ts

```
@@ -0,0 +1,588 @@
+            'string',
+            'Has each attendee been advised of the required documents and health requirements needed to leave the province or country?',
+            true,
+            ['No', 'Yes'],
+          ),
+        },
+      ),
+    }
+  }
+}
+
+// Maintenance Services
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)